src/renderer/secrets/Gopass.ts

```diff
@@ -22,19 +22,23 @@ export default class Gopass {
     }
 
     public static async history(key: string): Promise<HistoryEntry[]> {
-        return (await Gopass.execute(`history ${key}`))
-            .split(lineSplitRegex)
-            .filter(isDefined)
-            .map(historyLine => {
-                const lineSplit = historyLine.split(' - ')
-
-                return {
-                    hash: lineSplit[0],
-                    author: lineSplit[1],
-                    timestamp: lineSplit[2],
-                    message: lineSplit[3]
-                }
-            })
+        try {
+            return (await Gopass.execute(`history ${key}`))
+                .split(lineSplitRegex)
+                .filter(isDefined)
+                .map(historyLine => {
+                    const lineSplit = historyLine.split(' - ')
+
+                    return {
+                        hash: lineSplit[0],
+                        author: lineSplit[1],
+                        timestamp: lineSplit[2],
+                        message: lineSplit[3]
+                    }
+                })
+        } catch (e) {
```

Comment:
Thanks for the PR 👍

If we don’t take care about the exception I would implement it like:

```
} catch {
    return []
}
```